frontend/pendingTranslations.ftl

```diff
@@ -196,18 +196,18 @@ whatsnew-feature-tracker-removal-heading = Introducing email tracker removal
 # When translating, please make sure the resulting string is of roughly similar
 # length as the English version.
 whatsnew-feature-tracker-removal-snippet = Now { -brand-name-relay } can remove common email trackers from emails forwarded…
-whatsnew-feature-tracker-removal-description = Now { -brand-name-relay } can remove common email trackers from emails forwarded to you, helping you stay invisible to advertisers.
+whatsnew-feature-tracker-removal-description-2 = Now { -brand-name-relay } can remove common email trackers from emails forwarded to you, helping you stay invisible to advertisers. Turn it on in “Settings.”
 
 profile-stat-learn-more = Learn more
 profile-stat-learn-more-close = Close
 # This is displayed in small under a number in a large font indicating the number of trackers that have been removed from all emails sent to all of a user's masks
 profile-stat-label-trackers-removed = Trackers removed
 profile-stat-label-trackers-learn-more-part1 = Enabling tracker removal will remove common email trackers from your forwarded emails.
-profile-stat-label-trackers-learn-more-part2 = Important: Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images.
+profile-stat-label-trackers-learn-more-part2-2 = Important: Removing trackers may cause your email to look broken, because the trackers are often contained within images and links.
 # This is displayed in small under a number in a large font indicating the number of trackers that have been removed from all emails sent to a particular mask
 profile-label-trackers-removed = Trackers removed
 profile-trackers-removed-tooltip-part1 = With tracker removal enabled, common email trackers will be removed from your forwarded emails.
-profile-trackers-removed-tooltip-part2 = Important: Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images.
+profile-trackers-removed-tooltip-part2-2 = <b>Important:</b> Removing trackers may cause your email to look broken, because the trackers are often contained within images and links.
```

Comment:
non-blocking: This is new to me - I did not know we could include html within strings.

Reply:
Yeah, it's kinda cumbersome at the moment since you can't use the `useString` hook - you'll need to use `<Localized>`. (Incidentally, I have [a pull request](https://github.com/projectfluent/fluent.js/pull/595) open on `@fluent/react` to provide a similar hook API for it.) It's also not exactly HTML; I could've used anything tag-like here (e.g. `<notice>Important:</notice>`, and then in the `elems` attribute of `<Localized>` you can provide a mapping of that to actual tags.